### PR TITLE
PDCT 546 vespa control

### PR DESCRIPTION
**Pull request by PatFawbertMills**
Add 'secret' button to control whether to use the vespa API, it simply adds `?use_vespa=true` to the query string

### CODE REVIEW

File: src/hooks/useSearch.ts [hunk 1]
```diff
@@ -20,12 +20,14 @@ async function getSearch(query = initialSearchCriteria) {
     },
   };
 
+  // TODO: remove this later when BE is updated
+  const url = `/searches${query.use_vespa ? "?use_vespa=true" : ""}`;
```
Comment (THOR300) [inline src/hooks/useSearch.ts:24]: Does this need to be `True` with an uppercase T?

**Review by THOR300 — COMMENTED** [review]
Small question!

File: src/hooks/useSearch.ts (hunk 1 above)
Comment (PatFawbertMills) [reply src/hooks/useSearch.ts:24]: No, lower case was working fine when I tested this

**Review by diversemix — APPROVED** [review]
lgtm

**Pull request merged**